### ISSUE TITLE
CMake Tools does not seem to be installing the newer version

### ISSUE THREAD

**Issue by sursu** [microsoft/vscode-cmake-tools#645]
### Brief Issue Summary

Every single time I start VS Code I get the following notification:

```
There is a new version of CMake available. You are running 3.10.2, and 3.13.2 is 
available. 
Would you like CMake Tools to download and install this update automatically?
```
Every time I press `Yes`.
Then, for a brief moment it says: "Downloading cmake-3.13.2..." but nothing more appears to be happening.

Is it even being installed?

### Platform and Versions

- **Operating System**: Ubuntu 18.04
- **CMake Version**: 3.10.2
- **VSCode Version**: 1.30.2
- **CMake Tools Extension Version**: 1.1.3
- **Compiler/Toolchain**: GCC 7.3

**Comment by sursu** [comment]
Seems to be linked with [#622](https://github.com/vector-of-bool/vscode-cmake-tools/issues/622)

**Comment by Zingam** [comment]
Doesn work for me either. I have installed my own CMake. Also it be nice if the extension just installs its own version of Cmake if there is none installed.

**Comment by hgroene** [comment]
Same problem here

**Comment by dennisclausen** [comment]
Still same for me. Kinda annoying.

**Comment by bobbrow** [comment]
The auto-update feature is being removed from the next version of CMake Tools.

**Comment by bobbrow** [comment]
This issue has been addressed in CMake Tools 1.2.0 which is available in the VS Code Marketplace or on the [Releases Page](https://github.com/microsoft/vscode-cmake-tools/releases/tag/v1.2.0)

Because the publisher id has changed, VS Code does not automatically upgrade you yet. The Marketplace team will enable this in the next few days.  If you want to install 1.2.0 now, please make sure you uninstall any earlier versions of CMake Tools manually to avoid any conflicts.
